### DISCUSSION TITLE
Adding Tinybird Datasource

### DISCUSSION THREAD

**Discussion by kjohnstonadept** [getredash/redash#7116] [Q&A] [answered]
I just installed redash using this AMI: 

https://console.aws.amazon.com/ec2/home?region=us-east-1#LaunchInstanceWizard:ami=ami-0d915a031cabac0e0

I don't see how to add a Tinybird datasource.

**Comment by kjohnstonadept** [comment, marked as answer]
I figured this out - you need to use the preview version of redash to get Tinybird support. The AMI and the default docker image used in the setup script does not have support. Change PREVIEW in setup.sh to yes and re-install and it will show up.

**Reply by justinclift** [reply]
There's a `--preview` option to the setup.sh script to control that, but yeah manually changing the `PREVIEW` value in the file will get it happening too. :smile:
